librz/core/cdebug.c

```diff
@@ -793,3 +793,105 @@ RZ_API bool rz_core_debug_step_skip(RzCore *core, int times) {
 	}
 	return true;
 }
+
+/**
+ * \brief Free RzBacktrace
```

Comment:
You can just remove this doxygen, I think, it's unnecessary. Free functions are self-documenting if there aren't special API requirements.